### DISCUSSION TITLE
Welcome to OmniGibson Discussions!

### DISCUSSION THREAD

**Discussion by cgokmen** [StanfordVL/BEHAVIOR-1K#945] [Announcements]
This is where to post questions about OmniGibson and BEHAVIOR-1K.

**Comment by aoloo-r** [comment]
I get this error. Does it mean that this file is no longer available in the datsets? FileNotFoundError: [Errno 2] No such file or directory: OmniGibson/omnigibson/data/og_dataset/scenes/Ihlen_0_int/json/Ihlen_0_int_task_store_firewood_0_0_template.json

**Reply by ChengshuLi** [reply]
For each task (such as `store_firewood`), we currently only provide one pre-sampled initial configuration. Specifically for `store_firewood`, we have `Merom_0_int/json/Merom_0_int_task_store_firewood_0_0_template.json`. 

However, you can still online sample a task in any scene of your choice, with `online_sampling=True` in `BehaviorTask` and specifying the scene name to `Ihlen_0_int`, for instance. 

I will lock this conversation. All future discussions should open new tickets, rather than commenting on this announcement post. Thanks!
